usr/src/test/zfs-tests/tests/functional/zvol/zvol_swap/zvol_swap_004_pos.ksh

```diff
@@ -61,23 +72,34 @@ for vbs in 8192 16384 32768 65536 131072; do
 		log_must $ZFS create -s -b $vbs -V $volsize $vol
 		log_must $SWAP -a $swapname
 
-		if ((mem <= min)); then		# volsize should be 2G
-			new_volsize=$(get_prop volsize $vol)
-			((new_volsize == min)) || log_fail \
-			    "Unexpected volsize: $new_volsize"
-		elif ((mem >= max)); then	# volsize should be 16G
-			new_volsize=$(get_prop volsize $vol)
-			((new_volsize == max)) || log_fail \
-			    "Unexpected volsize: $new_volsize"
-		else				# volsize should be 'mem'
-			new_volsize=$(get_prop volsize $vol)
-			((new_volsize == mem)) || log_fail \
-			    "Unexpected volsize: $new_volsize"
-		fi
+                if ((mem <= min)); then         # volsize should be 2G
```

Comment:
This whole section should be reverted. When new_volsize matches (which should happen on the first iteration) we'll do a "break 2" and skip the rest of the tests. The original code here is correct.